### PR TITLE
SVT-HEVC: Remove shared libs

### PR DESCRIPTION
**Pull request by 1480c1**

### CODE REVIEW

**Comment by wiiaboo** [comment]
Cool. You can remove it from ffmpeg_options_full_shared too and do_changeFFmpegConfig removals for bothstatic, as well.

File: media-autobuild_suite.bat [hunk 1]
```diff
@@ -98,8 +98,7 @@ libmodplug librtmp librubberband libssh libtesseract libxavs libzmq ^
 libzvbi openal libvmaf libcodec2 libsrt ladspa #vapoursynth #liblensfun #librav1e
 
 :: options also available with the suite that add shared dependencies
-set ffmpeg_options_full_shared=opencl opengl libsvthevc ^
-cuda-nvcc libnpp libopenh264
+set ffmpeg_options_full_shared=opencl opengl cuda-nvcc libnpp libopenh264
```
Comment (wiiaboo) [inline media-autobuild_suite.bat]: Don't forget to readd it to _full.

**Comment by 1480c1** [comment]
Will be away for about 2 hours. Getting braces off

**Comment by 1480c1** [comment]
Still in the orthodontist, but have a little bit of time

**Comment by wiiaboo** [comment]
LGTM.

**Pull request merged**